### ISSUE TITLE
Nami Wallet imports a new address when I input seed phrases 

### ISSUE THREAD

**Issue by aidansmithh** [Berry-Pool/nami-wallet#223]
I have recently upgraded to a new laptop. I installed Nami wallet onto the new laptop and clicked import wallet. I put in my seed phrases and it's opened a new wallet with none of my ADA or assets in. 

Confused I logged into CNFT.IO and put some of my assets up for sale because it was linked with the original address. When I sold my assets I couldn't send them because my nami wallet was saying there is no assets in the new wallet address. 

Trying to fix the problem I stupidly disconnected the original wallet address from CNFT.IO and now I can't get into the wallet with all the assets in because the seed phrases open the wallet with no assets in... 

I am positive the seed phrases are correct because I write all the see phrases for Daedalus, yoroii and nami on the same piece of paper.

I have noticed a few other people are having this issue? would greatly appreciate any help!

**Comment by Sakura-Digitals** [comment]
The 24 word phrase import in Nami is entered from the top left to bottom left (1-12) then from top right to bottom right (12-24).

**Comment by aidansmithh** [comment]
Yeah I know, I have a 12 worded seed phrase anyway. The problem is the seed phrase has somehow opened a new wallet with a new address

**Comment by alessandrokonrad** [comment]
Possible you have created sub accounts in Nami?

**Comment by aidansmithh** [comment]
so an account within my main account?

**Comment by aidansmithh** [comment]
does a sub account give you a new wallet address?

**Comment by aidansmithh** [comment]
> Possible you have created sub accounts in Nami?

Hey Alessandro, I haven't created sub accounts. I only ever used the one account
